helix-core/src/main/java/org/apache/helix/controller/rebalancer/topology/Topology.java

```diff
@@ -55,16 +55,10 @@
   private final List<String> _liveInstances;
   private final Map<String, InstanceConfig> _instanceConfigMap;
   private final ClusterConfig _clusterConfig;
-  private final boolean _topologyAwareEnabled;
+  private static final String DEFAULT_PATH_PREFIX = "Helix_default_";
```

Comment:
Helix usually have all capitalized strings as our naming convention. Is this newly introduced or exists before?

Reply:
TFTR.

I did not change the prefix. Please refer to line 105 in the old code.

104          for (String type : _types) {
105            _defaultDomainPathValues.put(type, "Helix_default_" + type);
106            lastType = type;
107          }